Apps/phexpanse/expanse.json

```diff
@@ -0,0 +1,519 @@
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Issuer",
+                    "column_order": 7
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.certificate.subjectName",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Subject",
+                    "column_order": 8
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.certificate.md5Hash",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "md5 Hash",
+                    "column_order": 9
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.certificate.pemSha1",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "pemSha1",
+                    "column_order": 10
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.certificate.pemSha256",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "pemSha256",
+                    "column_order": 11
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.certificate.publicKeyAlgorithm",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Public Key Algo",
+                    "column_order": 12
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.certificate.publicKeyBits",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Public Key Length",
+                    "column_order": 13
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.certificateAdvertisementStatus.0",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Certificate Advertisement Status",
+                    "column_order": 14
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "lookup behavior",
+            "identifier": "lookup_behavior",
+            "description": "Returns Expanse Behavior data for an ip. Limited to 30 flows within the last 30 days",
+            "type": "investigate",
+            "read_only": false,
+            "parameters": {
+                "ip": {
+                    "description": "Internal IP address to search for",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
```

Comment:
`primary` should be false here because there are no values in the `contains` list

Reply:
Thanks @alexa-phantom, this is a good catch, in this instance `contains` should include `ip`. I've updated.